scrapers/MilfVR.yml

```diff
@@ -0,0 +1,24 @@
+name: "MilfVR"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - milfvr.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    common:
+      $info: //div[@class="detail"]
+    scene:
+      Title: //div[@class="detail__header detail__header-lg"]/h1
+      Date:
+        selector: $info//span[@class="detail__date"]/text()
+        parseDate: 2 January, 2006
+      Details:
+        selector: //div[@class="detail__txt detail__txt-show_lg"]/text()|//span[@class="more__body"]/text()
+        concat: " "
+      Tags:
+        Name: $info//div[@class="tag-list__body"]//a/text()
+      Performers:
+        Name: //div[@class="detail__inf detail__inf-align_right"]/div[@class="detail__models"]/a/text()
+      Image: //div[@class="photo-strip__body"]/div[@class="photo-strip__slide"]/img/@src
```

Comment:
the above gets a lot of images and only the smaller res ones
`(//div[@class="photo-strip__body"]/div[@class="photo-strip__slide"])[1]/@data-src` seems to work  better

Reply:
You're right.  I tested it out and this approach is a lot better.  I've learned something thanks to your feedback.  Fixed.